### PR TITLE
Added choices attribute to unit field under sensor

### PR DESCRIPTION
**Pull request by Ethan-bradley**
This fixes issue #13, which was to add a dropdown field for the measurements section in the sensor field.

### CODE REVIEW

**Comment by r-b-g-b** [comment]
Thanks Ethan! The Travis error says that a database migration file is missing. If this change requires a database migration, could you commit that as well?

File: woeip/apps/air_quality/models.py [hunk 1]
```diff
@@ -31,8 +31,11 @@ class Sensor(models.Model):
     """A sensor is something that measures something, i.e., it produces a single measurement value
     at a time.
     """
+
     name = models.CharField(max_length=256)
-    unit = models.CharField(max_length=256,
+    Unit_Choices = (('mg/m3','mg/m3'),('ppm','ppm'),('g/m3','g/m3'),('PM10','PM10'),('PM2.5','PM2.5'),('ug/m3','ug/m3'))
```
Comment (r-b-g-b) [inline woeip/apps/air_quality/models.py]: Let's use PEP8 and flake8 style conventions:
- ``Unit_Choices`` --> ``unit_choices`` (https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)
- spaces after commas (https://lintlyci.github.io/Flake8Rules/rules/E231.html)

We can also use unicode characters, so ``u`` --> ``μ``

**Review by TangoYankee — CHANGES_REQUESTED** [review]
Before we perform a full review, please address the failing tests.
* `Makefile:43: recipe for target 'quality' failed
make: *** [quality] Error 1`
* `Migration files are missing. Please run the makemigrations management command, and commit the migrations.
Makefile:53: recipe for target 'detect_missing_migrations' failed`

**Review by r-b-g-b — CHANGES_REQUESTED** [review]

**Comment by TangoYankee** [comment]
Change PR to eb/choices_attributes

**Pull request closed without merging**